python/semantic_kernel/connectors/ai/open_ai/models/chat/tool_calls.py

```diff
@@ -21,3 +23,26 @@ def __add__(self, other: Optional["ToolCall"]) -> "ToolCall":
             type=self.type or other.type,
             function=self.function + other.function if self.function else other.function,
         )
+
+    def parse_arguments(self) -> Dict[str, str]:
```

Comment:
The function can already has this function, why not let that handle it? Function call can be kept for the inner function and translating it into a kernel_function rather then a openai function

Reply:
I see. I was thinking that `FunctionCall` would go away, but I guess we'll keep it to handle the inner portion of a tool_call.